tests/test_box3d.py

```diff
@@ -5,13 +5,70 @@
+                               LiDARInstance3DBoxes, bbox3d2roi,
+                               bbox3d_mapping_back)
 from mmdet3d.core.bbox.structures.utils import (get_box_type, limit_period,
                                                 points_cam2img,
                                                 rotation_3d_in_axis,
                                                 xywhr2xyxyr)
 
 
+def test_bbox3d_mapping_back():
+    bboxes = BaseInstance3DBoxes(
+        [[
+            -5.24223238e+00, 4.00209696e+01, 2.97570381e-01, 2.06200000e+00,
+            4.40900000e+00, 1.54800000e+00, -1.48801203e+00
+        ],
+         [
+             -2.66751588e+01, 5.59499564e+00, -9.14345860e-01, 3.43000000e-01,
+             4.58000000e-01, 7.82000000e-01, -4.62759755e+00
+         ],
+         [
+             -5.80979675e+00, 3.54092357e+01, 2.00889888e-01, 2.39600000e+00,
+             3.96900000e+00, 1.73200000e+00, -4.65203216e+00
+         ],
+         [
+             -3.13086877e+01, 1.09007628e+00, -1.94612112e-01, 1.94400000e+00,
+             3.85700000e+00, 1.72300000e+00, -2.81427027e+00
+         ]])
+    new_bboxes = bbox3d_mapping_back(bboxes, 1.1, True, True)
+    expected_new_bboxes = torch.tensor(
+        [[-4.7657, 36.3827, 0.2705, 1.8745, 4.0082, 1.4073, -1.4880],
+         [-24.2501, 5.0864, -0.8312, 0.3118, 0.4164, 0.7109, -4.6276],
+         [-5.2816, 32.1902, 0.1826, 2.1782, 3.6082, 1.5745, -4.6520],
+         [-28.4624, 0.9910, -0.1769, 1.7673, 3.5064, 1.5664, -2.8143]])
+    assert torch.allclose(new_bboxes.tensor, expected_new_bboxes, atol=1e-4)
+
+
+def test_bbox3d2roi():
+    bbox_0 = torch.tensor([[
+        -5.24223238e+00, 4.00209696e+01, 2.97570381e-01, 2.06200000e+00,
+        4.40900000e+00, 1.54800000e+00, -1.48801203e+00
+    ],
+                           [
+                               -5.80979675e+00, 3.54092357e+01, 2.00889888e-01,
+                               2.39600000e+00, 3.96900000e+00, 1.73200000e+00,
+                               -4.65203216e+00
+                           ]])
+    bbox_1 = torch.tensor([[
+        -2.66751588e+01, 5.59499564e+00, -9.14345860e-01, 3.43000000e-01,
+        4.58000000e-01, 7.82000000e-01, -4.62759755e+00
+    ],
+                           [
+                               -3.13086877e+01, 1.09007628e+00,
+                               -1.94612112e-01, 1.94400000e+00, 3.85700000e+00,
+                               1.72300000e+00, -2.81427027e+00
+                           ]])
+    bbox_list = [bbox_0, bbox_1]
+    rois = bbox3d2roi(bbox_list)
+    expected_rois = torch.tensor(
+        [[0.0000, -5.2422, 40.0210, 0.2976, 2.0620, 4.4090, 1.5480, -1.4880],
+         [0.0000, -5.8098, 35.4092, 0.2009, 2.3960, 3.9690, 1.7320, -4.6520],
+         [1.0000, -26.6752, 5.5950, -0.9143, 0.3430, 0.4580, 0.7820, -4.6276],
+         [1.0000, -31.3087, 1.0901, -0.1946, 1.9440, 3.8570, 1.7230, -2.8143]])
+    assert torch.allclose(rois, expected_rois, atol=1e-4)
```

Comment:
We can simply use boxes like [[-5.2422, 40.0210, 0.2976, 2.0620, 4.4090, 1.5480, -1.4880], ......] for boxes0 and boxes1 and use `torch.eq` here.